README.md

```diff
@@ -86,18 +122,6 @@ module "my-fargate-task" {
       ]
     }
   )
-  container_environment_variables = [
-    {
-      name  = "FOO"
-      value = "bar"
-    }
-  ]
-  container_secrets = [
-    {
-      name      = "FOO_SECRET"
-      valueFrom = "arn:aws:ssm:${local.region}:${local.account_id}:parameter/foo_secret"
-    }
-  ]
```

Comment:
I kinda like keeping the environment variables in the example for some reason but I guess it's not necessary?

Reply:
these properties do not exist anymore, they are now part of the container definition itself.

Reply:
Great reason to not that them in there